MSAL/src/MSALPublicClientApplication.m

```diff
@@ -676,15 +676,6 @@ - (void)acquireTokenForScopes:(NSArray<NSString *> *)scopes
     else if (!_redirectUri.brokerCapable)
     {
         interactiveRequestType = MSIDInteractiveRequestLocalType;
-
-#if DEBUG && TARGET_OS_IPHONE
-        // Unless broker is explicitly disabled, show the warning in debug mode to configure broker correctly
-        NSURL *redirectUri = [MSALRedirectUriVerifier defaultBrokerCapableRedirectUri];
-        NSString *brokerWarning = [NSString stringWithFormat:@"The configured redirect URI for this application doesn't support brokered authentication. This means that your users might experience worse SSO rate or not be able to complete certain conditional access policies. To resolve it, register %@ scheme in your Info.plist and add \"msauthv2\" under LSApplicationQueriesSchemes. Go to \"aka.ms/msalbroker\" to check possible steps to resolve this warning", redirectUri.scheme];
-
-        MSID_LOG_WARN(nil, @"%@", brokerWarning);
-        NSLog(@"%@", brokerWarning);
-#endif
```

Comment:
why do we want to remove the warning? :)

Reply:
Because we want to continue working silently, when redirect uri is not compatible with broker.